packages/jade-coffee/package.js

```diff
@@ -1,19 +1,21 @@
 Package.describe({
   summary: "Jade template language with inline coffeescript and Javascript support",
-  version: "0.1.4",
+  version: "0.1.6",
   name: "xiphy:jade-coffee",
   git: "https://github.com/xiphias/meteor-jade-coffee.git",
   documentation: "../../README.md"
 });
 
 Package.registerBuildPlugin({
-  name: "compileJade",
+  name: "compileJadeBatch",
```

Comment:
Interesting, as neither version xiphy:master nor albinogeek:master will run entirely properly on my applications (giving the same "coffee run before jade ready" behavior.)  However; no build errors on Windows 10 with Meteor 1.2.1 .